featuretools/tests/feature_function_tests/test_transform_features.py

```diff
@@ -617,6 +617,18 @@ def test_cum_sum_use_previous_and_where_absolute(es):
         assert v == cvalues[i]
 
 
+def test_cum_handles_uses_full_entity(es):
+    log_value_feat = es['log']['value']
+    for primitive in [CumSum, CumMean, CumMax, CumMax]:
```

Comment:
`CumMax` is in the list twice and `CumCount` and `CumMin` are not present.

Reply:
fixed